docs/_includes/attr-doc.adoc

```diff
@@ -112,6 +126,65 @@ Another approach is to change the order of substitutions that are applied where
 The application is called {app-name}.
 ----
 
+==== Splitting attribute entry values over multiple lines
+
+===== Splitting long value strings
+
+Use backslashes `\` at the end of each attribute value to split long listings.
+
+[example]
+.A long, single line attribute
+[source,asciidoc]
+----
+:long_verbose: If you have a very long line of text that you need to substitute regularly in a document, you may find it easier to split it neatly in the header so it remains readable to the next person reading your docs code.
+----
+
+To split the block up and make it more readable, insert a  forward-slash (`\`) at the end of the line to extend the value to the next line.
+
+[example]
+.A long, logically-split attribute
+[source]
+----
+:long_verbose: If you have a very long line of text \
+that you need to substitute regularly in a document, \
+you may find it easier to split it neatly in the header \
+so it remains readable to folks reading your docs code.
+----
+
+NOTE: You can leave long lines as they are. Splitting attribute entry values such as the preceding example is only suggested for docs code readability.
+
+===== Preserving endlines
+
+Endlines are removed when the document is processed, unless a plus symbol surrounded by spaces (`+ \`) appears before the trailing backslash.
+
+[example]
+.When endlines matter
+[source]
+----
+:haiku: Write your docs in text, + \
+AsciiDoc makes it easy, + \
+Now get back to work! + \
+----
+
+Preserving endlines allows you to split an attribute value across multiple lines.
```

Comment:
It would be more accurate to say:

This syntax ensures that the newlines are preserved in the output document as hard line breaks.